packages/esm-patient-search-app/src/patient-search-page/patient-search-page.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { isDesktop } from '@openmrs/esm-framework';
+import PatientSearchPageComponent from './patient-search-page.component';
+
+const mockIsDesktop = isDesktop as jest.Mock;
+
+jest.mock('@openmrs/esm-framework', () => ({
+  ...jest.requireActual('@openmrs/esm-framework'),
+  isDesktop: jest.fn(),
+}));
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useParams: jest.fn(() => ({
+    page: 1,
+  })),
+  useLocation: jest.fn(),
+  useSearchParams: jest.fn(() => [
+    {
+      get: jest.fn(() => 'John'),
+    },
+  ]),
+}));
```

Comment:
Do we have to mock this?

Reply:
yes, it was throwing type error undefined.